### DISCUSSION TITLE
How do I add a second Y-Axis on a plot?

### DISCUSSION THREAD

**Discussion by quangnguyen09** [cetz-package/cetz-plot#56] [Q&A] [answered]
Looking through the manual, I can see there are mentioned of X2 and Y2 axis. However, I am unable to figure out how to add those axis to my plot. How do I enable these axis on my plot?

**Comment by johannes-wolf** [comment, marked as answer]
Additional axes automatically get added as soon as you use them via the `axes: (...)` parameter (for `plot.add` and `plot.add-*` commands). The only visible additional axes are the `x2` and `y2` with the `scientific` axis-style. But you can use any number of axes.

**Reply by quangnguyen09** [reply]
The `axes: (...)` parameter was the answer. Thank you!
